Analysis/Tasks/PWGHF/taskDPlus.cxx

```diff
@@ -54,23 +59,28 @@ struct TaskDPlus {
     for (auto& candidate : candidates) {
       hmass->Fill(InvMassDPlus(candidate));
       hptcand->Fill(candidate.pt());
+      hEta->Fill(candidate.eta());
+      hCt->Fill(CtDPlus(candidate));
+      hdeclength->Fill(candidate.decayLength());
+      hdeclengthxy->Fill(candidate.decayLengthXY());
+      hnormdlxy->Fill(candidate.decayLengthXYNormalised());
+      hCPA->Fill(candidate.cpa());
+      hCPAxy->Fill(candidate.cpaXY());
+      himpparxy->Fill(candidate.impParXY());
+      hnormip->Fill(candidate.normIP());
+      himpparsumsq->Fill(candidate.impParProngSqSum());
+      hDecLenErr->Fill(candidate.errorDecayLength());
+      hDecLenXYErr->Fill(candidate.errorDecayLengthXY());
+      hImpParErr->Fill(candidate.errorImpactParameter0());
+      hImpParErr->Fill(candidate.errorImpactParameter1());
+      hImpParErr->Fill(candidate.errorImpactParameter2());
```

Comment:
Why are the histograms reshuffled?

Reply:
This order just makes more sense to me, no specific reason.